sockeye/evaluate.py

```diff
@@ -54,8 +54,7 @@ def raw_corpus_chrf(hypotheses: Iterable[str], references: Iterable[str]) -> flo
     :param references: Reference stream.
     :return: chrF score as float between 0 and 1.
     """
-    return sacrebleu.corpus_chrf(hypotheses, references, order=sacrebleu.sacrebleu.CHRF_ORDER, beta=sacrebleu.sacrebleu.CHRF_BETA,
-                                 remove_whitespace=True).score
+    return sacrebleu.corpus_chrf(hypotheses, [references]).score
```

Comment:
I know we fix the version of sacrebleu in the requirements, but I think its still worth making the settings here explicit by passing the keyword arguments.

Reply:
Sure, I can add them back in. I thought that is redundant since those values are taken from sacrebleu anyway.

Reply:
yep, but they can change it and maybe forget to update the version, or we at some point don't fix the version anymore :)

Reply:
Then wouldn't it be better to use actual numbers instead of a new way to access those variables in sacrebleu? (e.g. `sacrebleu.metrics.CHRF.ORDER`)

Reply:
True, I was only looking at the `remove_whitespace` flag really. Let's leave as you have it for now.